topostats/tracing/pruning.py

```diff
@@ -11,13 +11,13 @@
 
 from topostats.logs.logs import LOGGER_NAME
 from topostats.tracing.skeletonize import getSkeleton
-from topostats.tracing.tracingfuncs import genTracingFuncs
+from topostats.tracing.tracingfuncs import coord_dist, genTracingFuncs, order_branch
 from topostats.utils import convolve_skeleton
 
 LOGGER = logging.getLogger(LOGGER_NAME)
 
 
-def prune_skeleton(image: npt.NDArray, skeleton: npt.NDArray, **kwargs) -> npt.NDArray:
+def prune_skeleton(image: npt.NDArray, skeleton: npt.NDArray, px_2_nm: float, **kwargs) -> npt.NDArray:
```

Comment:
I'd be inclined to keep the argument names the same as used elsewhere so its consistent across functions and there is no mental overhead in equating `px_2_nm` to `pixel_to_nm_scaling`. Variables are scoped correctly do we don't need to come up with endless variations to avoid conflicts. There is also a definite advantage to keeping names consistent too as it can make using `kwargs` very easy if dictionaries contain settings.

```suggestion
def prune_skeleton(image: npt.NDArray, skeleton: npt.NDArray, pixel_to_nm_scaling: float, **kwargs) -> npt.NDArray:
```

**NB** - This is a good case where you can use the Language Server of VSCode to replace all instances. In my experience it works very well, the one thing it _doesn't_ do is update the renaming within docstrings so that has to be changed manually.

Reply:
cool, done! Also mine seems to have renamed docstrings - or at least my process of `cmd`+`shift`+`h` did :)